src/components/app/App.tsx

```diff
@@ -49,7 +49,7 @@ export class Evergreen extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      APIClient: rest.EvergreenClient("", "", "", "", true),
+      APIClient: rest.EvergreenClient("https://evergreen.mongodb.com/api", "https://evergreen.mongodb.com")
```

Comment:
This should come from the config

Reply:
You're right- I misunderstood this ticket initially and thought all the config stuff should be gone too. I've fixed this now.